install_config/install/docker_registry.adoc

```diff
@@ -169,9 +169,41 @@ $ oadm registry --service-account=registry \
 ----
 endif::[]
 
-[[viewing-logs]]
 
+[[registry-ip-address]]
+=== Registry IP Address
+
+Because the integrated registry is referenced by its service IP address,
+if you decide to delete and recreate the docker-registry service,
+you should either take care to re-use its address,
+or reboot the cluster after creating the new docker-registry service.
```

Comment:
As mentioned in the bz, the only thing you need to restart is the master api. Restarting anything else will cause unnecessary downtime.